torchao/quantization/prototype/mixed_precision/scripts/sensitivity_study.py

```diff
@@ -0,0 +1,95 @@
+import torch
```

Comment:
might wanna call this file something else since you're about to do the real sensitivity analysis

Reply:
Removed this file for now and will commit the real sensitivity analysis  in milestone2